tests/main/firstboot/task.yaml

```diff
@@ -1,68 +0,0 @@
-summary: Check that firstboot assertions are imported and snaps installed
```

Comment:
I feel a bit uneasy about dropping the entire test. Pardon my ignorance on this but what in this branch breaks it?